Jasmine/Common/ViewControllers/GameElements/BaseGameViewControllerDelegate.swift

```diff
@@ -24,5 +24,5 @@ protocol BaseGameViewControllerDelegate: class {
     /// This is also the place to indicate when the user has won/lost.
     /// Note also that if the user has won/lost, the score in the `redisplay(_ newScore)` will be
     /// taken as the end game score. So update it before calling end game.
-    func notifyGameStatus()
+    func updateGameStatus()
```

Comment:
as this method will be implemented in the view controller, it is weird to have a method called "updateGameStatus" in a VC, should still name it such that it implies the VC to update the view when game status changed